osm2gtfs/creators/et_addisababa/trips_creator_et_addisababa.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+
+from datetime import timedelta, datetime
+
+from osm2gtfs.creators.trips_creator import TripsCreator
+from osm2gtfs.core.helper import Helper
+from osm2gtfs.core.elements import Line
+
+from transitfeed.trip import Trip
+
+ 
+def time_string_to_minutes(time_string):
+    (hours, minutes, seconds) = time_string.split(':')
+    return int(hours) * 60 + int(minutes)
+
+class TripsCreatorEtAddisababa(TripsCreator):
+    service_weekday = None
+
+    def add_trips_to_feed(self, feed, data):
+        self.service_weekday = feed.GetDefaultServicePeriod()
+        self.service_weekday.SetStartDate(
+            self.config['feed_info']['start_date'])
+        self.service_weekday.SetEndDate(self.config['feed_info']['end_date'])
+        self.service_weekday.SetWeekdayService(True)
+        self.service_weekday.SetWeekendService(True)
+
+        lines = data.routes
+        for route_ref, line in sorted(lines.iteritems()):
+            if not isinstance(line, Line):
+                continue
+            print("Generating schedule for line: " + route_ref)
+
+            flex_flag = None
+            if 'route_master' in line.tags and line.tags['route_master'] == "light_rail":
+                route_type = "Tram"
+                route_suffix = " (Light Rail)"
+            elif 'route_master' in line.tags and line.tags['route_master'] == "share_taxi":
+                route_type = "Bus"
+                route_suffix = " (Minibus)"
+                flex_flag = 0
```

Comment:
I don't like the idea of it being hard-coded. Can we rather have that information as an attribute on the route / route_master in OSM ?

Reply:
Possible, need to discuss with the OSM community how it is mapped best on OSM side.